### ISSUE TITLE
Fix model input error

### ISSUE THREAD

**Issue by phecda-xu** [Z-yq/TensorflowASR#10]
更新代码后尝试训练transducer模型时遇到如下报错：
```
 Cannot convert a list containing a tensor of dtype <dtype: 'int32'> to <dtype: 'float32'>
```
经过分析后发现问题出在这里：

实际需要的输入是这样
[call(self, features, predicted=None, training=False)](https://github.com/Z-yq/TensorflowASR/blob/848eaccc135617f3d9d138c6e1fb96748b064979/AMmodel/transducer_wrap.py#L165)

features是一个信号或者特征的tensor，predicted是label的tensor

给模型的输入是这样
[logits = self.model([wavs, pred_inp], training=True)](https://github.com/Z-yq/TensorflowASR/blob/848eaccc135617f3d9d138c6e1fb96748b064979/trainer/transducer_runners.py#L54)
[logits = self.model([features, pred_inp], training=True)](https://github.com/Z-yq/TensorflowASR/blob/848eaccc135617f3d9d138c6e1fb96748b064979/trainer/transducer_runners.py#L56)

此时的features 变成了一个list从而导致在mel_layer里对 tensor进行tanspose操作时报错，应该把中括号去掉。

**Comment by Z-yq** [comment]
感谢反馈，由于我debug结束后，忘记还原之前的输入格式了。
现已修复。
您的方法也是可以的。
